yt/yql/agent/yql_agent.cpp

```diff
@@ -122,8 +122,12 @@ class TYqlAgent
 
         auto clustersConfig = Config_->GatewayConfig->AsMap()->GetChildOrThrow("cluster_mapping")->AsList();
 
-        auto singletonsConfigString = SingletonsConfig_
-            ? ConvertToYsonString(*SingletonsConfig_)
+        auto SingletonsConfigDefaultLogging = CloneYsonStruct(SingletonsConfig_);
+        // Compressed logs is broken if plugin tries to open and write to them.
+        SingletonsConfigDefaultLogging->Logging = TLogManagerConfig::CreateDefault();
```

Comment:
Local variables should be in camelCase
https://github.com/ytsaurus/ytsaurus/blob/main/yt/styleguide/cpp.md#variables-and-enums